spec/datadog/tracing/contrib/httprb/instrumentation_spec.rb

```diff
@@ -43,9 +43,7 @@
 
     # LibFFI native thread
     allow(::HTTP::Connection).to receive(:new).and_wrap_original do |method, *args, &block|
-      ThreadHelpers.with_leaky_thread_creation(:ffi) do
-        method.call(*args, &block)
-      end
+      method.call(*args, &block)
     end
```

Comment:
This block only existed to apply the `ThreadHelpers`, so it should be removed. Leaving it behind I think is quite confusing because at least for me it's one of those situations where you go "Wait... Is this doing anything? Why is it here? Am I missing something?